CrowdStrike/ember-toucan-core
# Add pnpm-sync-dependencies-meta-injected to improve DX

## 🚀 Description

This PR follows @vitch 's `headless-form` PR to make local development a lot better.  Instead of copying over what's there, I'm instead using @NullVoxPopuli 's [package](https://github.com/NullVoxPopuli/pnpm-sync-dependencies-meta-injected) that does this for us.

With these changes, instead of doing `pnpm` linking or running a script for local development, run `pnpm start` from the root of the monorepo. When making a change, the packages will rebuild and your change will be displayed.

- https://github.com/CrowdStrike/ember-headless-form/pull/140

---

## 🔬 How to Test

- Pull this branch down
- Run `pnpm i`
- Run `pnpm start`
- Verify the docs and test apps start
- Make a change under `ember-toucan-core` and `ember-toucan-form`
- Verify the change shows up in the docs/test app

---

## 📸 Images/Videos of Functionality

N/A - developer improvement only
